spec/helpers/strategies_form_helper.spec.rb

```diff
@@ -0,0 +1,225 @@
+# frozen_string_literal: true
```

Comment:
Can you fix the indentation in this file? We use 2 spaces for tabbing :)

Reply:
I made the change. Could you please verify.